.gitignore

```diff
@@ -239,4 +239,4 @@ pip-selfcheck.json
 
 # ----------------------------------------------------------------------------------------------------------------------------------------------------
 
-*config.yaml
\ No newline at end of file
+config*.yaml
```

Comment:
This should be reverted as it better supported different config names previously and was only changed out of impulse.

Reply:
Looking at this again, I changed it to this so that config files would be gathered together. For example `config.dev.yaml` and `conf.prod.yaml` are sorted together, which is more clear than `dev.config.yaml` and the prod equivalent. Going to retain this change.